src/metabase/driver.clj

```diff
@@ -940,6 +940,17 @@
   dispatch-on-initialized-driver
   :hierarchy #'hierarchy)
 
+(defmulti truncate!
+  "Delete the current contents of `table-name`.
+  If something like a SQL TRUNCATE statement is supported, we use that, but may otherwise fall back to explicitly
+  deleting rows, or dropping and recreating the table.
+  Depending on the driver, the semantics on whether triggers are fired, AUTO_INCREMENT is reset etc, is allowed to vary.
+  The application assumes that the implementation can be rolled back if inside a transaction."
+  ;; TODO Update this once we're ready-to-merge
+  {:added "0.51.0", :arglists '([driver db-id table-name])}
```

Comment:
same 50 vs. 51 question. 50 is currently scheduled to go out in May.

Reply:
Had a brain fart because I shadowed the release the *created* the milestone, hence fencepost error.